classes/services/queryBuilders/BaseQueryBuilder.inc.php

```diff
@@ -37,7 +37,9 @@ protected function bootstrap() {
 
 		// Map valid OJS3 config options to Illuminate database drivers
 		$driver = strtolower(Config::getVar('database', 'driver'));
-		if (substr($driver, 0, 8) === 'postgres') {
+		if (strpos($driver, 'sqlsrv') !== false) {
+		    $driver = 'sqlsrv';
```

Comment:
(Watch out for space-based indents.)

Reply:
Yes, I will do my best to avoid those space-based indents, I saw the documentation about them after all the modifications we made to support SQL server.